planning/behavior_path_planner/include/behavior_path_planner/turn_signal_decider.hpp

```diff
@@ -34,6 +35,10 @@ using route_handler::RouteHandler;
 class TurnSignalDecider
 {
 public:
+  mutable bool intersection_turn_signal_ = false;
+  mutable bool approaching_intersection_turn_signal_ = false;
+  mutable double intersection_distance_ = std::numeric_limits<double>::max();
+  mutable Pose intersection_pose_point_ = Pose();
```

Comment:
Not function related.

Rarely member variable is made as `public`.
And if I understand the code correctly, the `flag` is only set within `TurnSignalDecider` class. Outside of class, it is only evaluated.

Therefore I recommend making the variable as `private` and create additional member function that can be call to "check the flag".

Reply:
@zulfaqar-azmi-t4 I fixed in this commit [8fdacdd](https://github.com/autowarefoundation/autoware.universe/pull/1706/commits/8fdacddb6d8cc9d5515c60f6eff9dab93f186729)